tests/stdlib/trandom.nim

```diff
@@ -37,23 +24,44 @@ proc main =
   doAssert sample("a") == 'a'
```

Comment:
make tests robust to algorithmic changes:
```nim
doAssert a[0] == 1
doAssert a[1] == 0
```
=>
```nim
  doAssert a in [[0,1], [1,0]]
```

(also, makes this pass for `-b:js` up to here)